src/SettingsPanel.svelte

```diff
@@ -46,30 +47,57 @@
         {/if}
         {#each $formatsStore as format, formatIdx (format.id)}
             <tr out:fade>
-                <td
-                    ><input
-                        type="text"
-                        class="form-control"
-                        bind:value={format.name}
-                        on:input={saveFormats}
-                    /></td
-                >
-                <td>
-                    <input
-                        type="text"
-                        class="form-control"
-                        bind:value={format.rule}
-                        on:input={saveFormats}
-                    />
-                </td>
-                <td>
-                    <input
-                        type="text"
-                        class="form-control"
-                        bind:value={format.mime}
-                        on:input={saveFormats}
-                    />
-                </td>
+                {#if !supportsClipboard()}
+                    <td
+                        ><input readonly="readonly"
```

Comment:
The name and rule should be editable. Only the mime type field should be readonly (on firefox). This is what it should look like:

<img width="785" alt="Screen Shot 2023-03-24 at 1 26 58 PM" src="https://user-images.githubusercontent.com/320904/227597712-7ee42339-68fc-4fe6-8455-364588183a78.png">